src/Resources/Resources/help/Get-AzPolicyExemption.md

```diff
@@ -112,6 +112,7 @@ Accept wildcard characters: False
 
 ### -IncludeDescendent
 Causes the list of returned policy exemptions to include all exemptions related to the given scope, including those from ancestor scopes and those from descendent scopes.
+This parameter doesn't work when the requested scope is a management group scope.
```

Comment:
The help message is generated [according to this](https://github.com/Azure/azure-powershell/blob/47c99e3c5c58876debb7a3303b739ce06d944982/src/Resources/ResourceManager/Implementation/Policy/GetAzurePolicyExemption.cs#L66-L67). Please update the resource string otherwise it will be overwritten next time.

Reply:
Updated it.